prog-o-meter.py

```diff
@@ -80,19 +80,41 @@ def canvas_layout(self):
     def button_layout(self):
         """Display a button with the text "1 more day!" on the canvas.
         
-        Creates and display a button with the text "1 more day!" with the function add_day() as callback function. If user have already reached their goal, the button is displayed, but is disabled. 
-       
+        Creates and display a button with the text "1 more day!" with the function add_day() as callback function. If user have already reached their goal, the button is displayed, but is disabled.
+        
         Attributes:
             add_day_button: A button with the text "1 more day!", which calls the function add_day
         """
         self.add_day_button = Tk.Button(self.root, text = "1 more day!", command = self.add_day)
         self.add_day_button.pack()
-        if self.days >= self.GOAL:        # Disable add_day_button if goal have been reached
+        if self.days >= self.GOAL:  # Disable add_day_button if goal have been reached
             self.add_day_button.config(state = "disabled")
+        """Display a button with the text "Whoops!" on the canvas.
+        
+            Creates and display a button with the text "Whoops!" with the function delete_day() as callback function. If number of days of progress is 0, the button is displayed, but is disabled.
+        
+            Attributes:
+            whoops_button: A button with the text "Whoops!", which calls the function delete_day
+                """
+        self.whoops_button = Tk.Button(self.root, text = "Whoops!", command = self.delete_day)
+        self.whoops_button.pack(pady = 5)
+        if self.days <= 0:  # Disable Whoops button if days of progress become <= 0
+            self.whoops_button.config(state = "disabled")
+        """Display a button with the text "Add Multiple Days" on the canvas.
+        
+            Creates and display a button with the text "Add Multiple Days" with the function add_multidays() as callback function. If user have already reached their goal, the button is displayed, but is disabled.
+        
+            Attributes:
+            add_multidays_button: A button with the text "Add Multiple Days", which calls the function add_multidays
+                """
+        self.add_multidays_button = Tk.Button(self.root, text = "Add Multiple Days", command = lambda: self.add_multidays(self.days, self.GOAL, self.filename, self.username))
+        self.add_multidays_button.pack(pady = 5)
+        if self.days >= self.GOAL:  # Disable add_multidays_button if goal have been reached

```

Comment:
since we already have one check of this condition, I think it would make more sense to add what is on line 113  right below line 91, and then get rid of line 112.

Reply:
If you do make this change, remember to update the comment on the if-statement on line 90, to also include that it disables add_multidays_button